packages/jest/src/generators/init/init.ts

```diff
@@ -223,23 +57,21 @@ function updateProductionFileSet(tree: Tree) {
   updateNxJson(tree, nxJson);
 }
 
-function addJestTargetDefaults(tree: Tree, hasPlugin: boolean) {
+function addJestTargetDefaults(tree: Tree) {
   const nxJson = readNxJson(tree);
 
   nxJson.targetDefaults ??= {};
   nxJson.targetDefaults['@nx/jest:jest'] ??= {};
 
-  if (!hasPlugin) {
-    const productionFileSet = nxJson.namedInputs?.production;
+  const productionFileSet = nxJson.namedInputs?.production;
 
-    nxJson.targetDefaults['@nx/jest:jest'].cache ??= true;
-    // Test targets depend on all their project's sources + production sources of dependencies
-    nxJson.targetDefaults['@nx/jest:jest'].inputs ??= [
-      'default',
-      productionFileSet ? '^production' : '^default',
-      '{workspaceRoot}/jest.preset.js',
-    ];
-  }
+  nxJson.targetDefaults['@nx/jest:jest'].cache ??= true;
```

Comment:
How come we removed the check for `@nx/jest/plugin`? I don't think we want the targetDefaults fo PCv3.

Reply:
The check is already done when invoking this function https://github.com/nrwl/nx/pull/21088/files#diff-cbef53cf0c3f36af94e1a3f3ad60780ad4c6f884c04f3800afe22f972cfb3efaR106-R110.